src/bin/uhyve.rs

```diff
@@ -56,6 +56,16 @@ struct Args {
 	#[clap(long)]
 	file_mapping: Vec<String>,
 
+	/// The path that should be used for temporary directories storing unmapped files.
+	///
+	/// This is useful for manually created tmpfs filesystems and for selecting
+	/// directories not managed by a temporary file cleaner, which can remove open files
+	/// manually. In most cases, mapping the guest path /root/ instead should be sufficient.
+	///
+	/// Defaults to /tmp.
+	#[clap(long)]
+	tempdir_path: Option<String>,
```

Comment:
Ah, I'd also rather call this `tempdir`. It should be clear from the context and the description, that this is a path. Shorter options are better options :wink: